bucket/mpv.json

```diff
@@ -24,18 +24,22 @@
     "suggest": {
         "youtube-dl": "youtube-dl"
     },
-    "notes": "To set up file type associations and AutoPlay handlers use https://github.com/rossy/mpv-install",
+    "notes": [
+        "- To set up file type associations and AutoPlay handlers run '$dir\\installer\\mpv-install.bat' as administrator.",
```

Comment:
Why dashes??

Reply:
Well, I copy it from mpv-git. :joy: I'll remove it.